app/screens/AppNavigator/screens/Dex/PoolSwap/PoolSwapScreen.tsx

```diff
@@ -84,19 +144,22 @@ export function PoolSwapScreen ({ route }: Props): JSX.Element {
 interface TokenForm {
   control: Control
   controlName: string
-  token: AddressToken
-  onMaxPress?: (amount: string) => void
+  token: DerivedTokenState
```

Comment:
Created new interface DerivedTokenState, to handle when token has no balance, AddressToken instance will be undefined.